style_guide/README.md

```diff
@@ -1,206 +1,307 @@
 `imap_processing` Style Guide
 =============================
 
-This document serves as a style guide for all `imap_processing` software
-development.  Any potential contribution to the `imap_processing` repository
-should be checked against this guide, and any violation of the guide should be
-fixed before the code is committed to the `main` or `dev` branches.
+This document serves as a style guide for all `imap_processing` software development.  Any potential contribution to the
+`imap_processing` repository should be checked against this guide, and any violation of the guide should be fixed before
+the code is committed to the `main` or `dev` branches.
 
 
 ## tl;dr
 
-The following is a short version of this style guide to be used as a quick
-reference.  Further details about each of these are provided in the guide.
+The following is a short version of this style guide to be used as a quick reference.  Further details about each of
+these items are provided below in the guide.
 
-1. Use a [forking workflow](#git-&-github-workflow) for git/GitHub contributions.
-2. Update the [`poetry` environment](#poetry-environment) when dependencies
-   change.
-3. Use PEP8 for [python coding](#python-coding), with a few exceptions.
-4. Use PEP257 and `numpydocs` for [docstring conventions](#api-documentation),
-   with a few exceptions.
-5. Use nominal semantic versioning for [version numbers](#versioning).
-6. Be mindful of committing credentials and other [sensitive information](#security).
-7. Use specific [tools and libraries](#tools-and-library-recommendations) when
-   able to.
+1. Use a [forking workflow](#git-&amp-github-workflow) for git/GitHub contributions.
+2. Use `PEP8` for [python coding conventions](#python-coding), with a few exceptions.
+3. Use `PEP257` and `numpydocs` for [docstring conventions](#api-documentation), with a few exceptions.
+4. Update the [`poetry` environment](#poetry-environment) when dependencies change.
+5. Be mindful of committing credentials and other [sensitive information](#security).
+6. Follow agreed-upon [naming conventions](#naming-conventions) where applicable.
+7. Use specific [tools and libraries](#tools-and-library-recommendations) where applicable.
+8. Use nominal semantic versioning for [version numbers](#versioning).
+9. Follow a specific [release workflow](#release-workflow) when making releases.
 
```

Comment:
Would it be appropriate to

Reply:
Yeah, now that #15 is merged and we have automated doc builds, I think it makes sense to add a note here about updating the necessary `.rst` files when applicable.  I'll add that.

Beyond that, and beyond the mention of `PEP257`/`numpydocs` style, was there something else we could mention?

Reply:
I think just adding to update or write the documentation as required is all that's needed right now, although I will maybe just write up a documentation specific style guide